### PR TITLE
Add loki

### PR DESCRIPTION
**Pull request by zachariahmiller**
## Description

Add loki

## Related Issue

Fixes #

Relates to #

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

**Comment by zachariahmiller** [comment]
@Michael-Kruggel's initial loki work. I'll be working on integrating in a followup PR

File: capabilities/loki/values/values.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# -- Overrides the chart's name
+nameOverride: loki
+# -- Overrides the chart's computed fullname
+fullnameOverride: loki
+# -- Overrides the chart's cluster label
+clusterLabelOverride: null
+# -- Image pull secrets for Docker images
+imagePullSecrets:
+  - name: private-registry
```
Comment (jeff-mccoy) [inline capabilities/loki/values/values.yaml]: Do we actually need this with zarf?

File: capabilities/loki/zarf.yaml [hunk 2]
```diff
@@ -2,16 +2,25 @@
...
+      - name: loki
+        url: https://grafana.github.io/helm-charts/
+        version: 2.9.2
+        namespace: loki
+        valuesFiles:
+          - ./values/values.yaml
+    images:
+      - docker.io/grafana/loki:2.4.2
+  - name: uds-loki-config
+    required: true
```
Comment (jeff-mccoy) [inline capabilities/loki/zarf.yaml]: Required actually isn't enforced on import so I've been removing it here and putting in the importing zarf package

**Review by jeff-mccoy — COMMENTED** [review]
just some small nits

File: capabilities/loki/zarf.yaml (hunk 2 above)
Comment (zachariahmiller) [reply capabilities/loki/zarf.yaml]: i removed the required: true for loki and neuvector

File: capabilities/loki/values/values.yaml (hunk 1 above)
Comment (zachariahmiller) [reply capabilities/loki/values/values.yaml]: No. Removed it.

**Review by jeff-mccoy — APPROVED** [review]
lgtm thx

**Pull request merged**